server/system/processEnv.js

```diff
@@ -0,0 +1,25 @@
+const ProcessUtils = require('../../common/processUtils')
+
```

Comment:
to me ProcessEnv should go under server/utils. It's not used only in system, but rather into several server modules

Reply:
ok